demisto_sdk/commands/lint/linter.py

```diff
@@ -1601,15 +1612,28 @@ def _get_docker_images_for_lint(
 
             self._check_native_image_flag(docker_image_flag)
 
-            if native_image := self._get_native_image_name_from_config_file(
-                docker_image_flag
-            ):
+            supported_section = docker_image_flag
+            if docker_image_flag == DockerImageFlagOption.NATIVE_TARGET.value:
+                supported_section = DockerImageFlagOption.NATIVE_DEV.value
```

Comment:
Why did you choose here the native dev? just in order to get into the 'if native_image' cond? 
Cause in the end the native image will be the docker_image_target image right? 

What happens if the script\integration isn't supported by the native:dev (supported by the native:ga only) but you still want to run it on the docker image target? is it a possible scenario?

Reply:
that's not a possible scenario, we only want to target native-dev supported images